moveit_ros/perception/mesh_filter/CMakeLists.txt

```diff
@@ -11,13 +11,7 @@ set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES VERSION "${${PROJECT_NAME}_V
 ament_target_dependencies(${MOVEIT_LIB_NAME}
   rclcpp
   moveit_core
-  image_transport
```

Comment:
Why are all these dependencies dropped? They still seem to be used, no?

Reply:
These dependencies were added during the porting of the point_cloud_octomap_updater. Later I found these dependencies were used in some files that were never added into compilation. These files are: `depth_self_filter_nodelet.cpp`, `depth_self_filter_nodelet.h`, `mesh_filter.cpp`, `transform_provider.cpp` and `transform_provider.h` under `moveit_ros/perception/mesh_filter/`. I submitted a new commit to delete them.